ui/lib/screens/projects/projects_screen.dart

```diff
@@ -3,6 +3,7 @@
  * SPDX-License-Identifier: MIT
  */
 
+import 'package:bom_bar_ui/screens/packages/name_filter.dart';
```

Comment:
I try to use relative imports for anything inside the project, and absolute imports for packages from outside the project.

PS: Test code needs to always use absolute imports, since it lives outside the `lib` directory.